cmd/aks-periscope/aks-periscope.go

```diff
@@ -22,25 +24,38 @@ func main() {
 		log.Printf("Failed to create CRD: %+v", err)
 	}
 
+	clusterType := os.Getenv("CLUSTER_TYPE")
+
+	storageAccount := os.Getenv("AZURE_BLOB_ACCOUNT_NAME")
+	log.Printf("Storage Account: %s", storageAccount)
 	collectors := []interfaces.Collector{}
 	containerLogsCollector := collector.NewContainerLogsCollector(exporter)
 	collectors = append(collectors, containerLogsCollector)
-	systemLogsCollector := collector.NewSystemLogsCollector(exporter)
-	collectors = append(collectors, systemLogsCollector)
 	networkOutboundCollector := collector.NewNetworkOutboundCollector(5, exporter)
 	collectors = append(collectors, networkOutboundCollector)
-	ipTablesCollector := collector.NewIPTablesCollector(exporter)
-	collectors = append(collectors, ipTablesCollector)
```

Comment:
💡 Not sure but, why is this copy pasted to down below? if not intentional lets aim for only new feature work to be effected.

* Intent of asking this question is to also make sure that we document that in case of `AzureArc` why the `IPtables` logs et. al. will not be required?

Reply:
I copied and pasted to down below because we would be turning off the systemLogs, IPTables, etc. collectors for the connected cluster type (i.e. when arc mode is enabled), as outlined in our design doc. So yes, this was intentional. I will get back to you on the reasoning as to why we do not intend on reusing these collectors.

Reply:
Update: I reverted these changes in my most recent PR update since David had another PR to expose the feature flag for collectors, diagnosers, and exporters